### ISSUE TITLE
UTF-8 Title Support in Graph

### ISSUE THREAD

**Issue by junder873** [org-roam/org-roam#681]
### Description

When I try to export a graph where titles contain some utf-8 characters (in my case, 'ö'), the graph shows an encoding error and will not be complete.

One note on reproduction, I am using graphviz-2.38 and neato (instead of dot). To verify that this was indeed the problem, I removed the offending character from titles and it immediately worked.

### Environment

Copy info below this line into issue:
- Emacs: GNU Emacs 26.3 (build 1, x86_64-w64-mingw32)
 of 2019-08-29

**Comment by progfolio** [comment]
Could you please paste the full output of `org-roam-diagnostics`?

**Comment by progfolio** [comment]
I'm unable to produce this with `org-roam-graph-executable` set to `"neato"`, graphviz V 2.44.0, and the following file `"/home/n/org-roam/20200521195259-oh_no.org"`:

```org 
#+TITLE: öh-nö
öh-nö!
```

If you don't mind, could you post the encoding error or a picture with an example of a broken graph?

**Comment by junder873** [comment]
Unfortunately that is all I get from the org-roam-diagnostics (I assume all I do is M-x and then `org-roam-diagnostics`, if there is something else I need to do please let me know, I am rather new to Emacs).

It might be due to a different version of graphviz?

Here is the error I receive. 
![Capture](https://user-images.githubusercontent.com/27790977/82626428-ced9fd80-9bb5-11ea-96fb-57cd7775e5f3.PNG)

**Comment by progfolio** [comment]
Executing `M-x org-roam-diagnostics` should present you with a prompt that says
```
I'm using the following Emacs framework:
```
After answering that question, it should pop up a buffer with diagnostic information.

If that does not work for you, you can paste the output of the following commands in this issue:
`M-x org-roam-version`
`M-x org-version`
You can switch to the `*Messages*` buffer to copy those if `org-roam-diagnostics` does not work.

**Comment by junder873** [comment]
Thank you for your help, I think this is the info you want.

Copy info below this line into issue:
- Emacs: GNU Emacs 26.3 (build 1, x86_64-w64-mingw32)
 of 2019-08-29- Framework:  Emacs: GNU Emacs 26.3 (build 1, x86_64-w64-mingw32)
 of 2019-08-29
- Org: Org mode version 9.3.6 (9.3.6-elpa @ c:/Users/junde/AppData/Roaming/.emacs.d/elpa/org-9.3.6/)
- Org-roam: 1.1.1

**Comment by jethrokuan** [comment]
@junder873 sorry for getting back so late, are you still seeing this issue?

I'm able to produce a graph with the title "öh-nö" as well. If you're still seeing an issue, can you paste the line for the encoding error. You should be able to find the generated `dot` file in your temp directory.

**Comment by aleneum** [comment]
I had similar issues with unicode characters but they could be traced to `Graphviz`. When the SVG creation fails (the temp file will be empty), the graph will not appear. I used `Graphviz 2.38` from [Chocolatey](https://chocolatey.org/packages/Graphviz). Even a graph with a single node wont compile when the label contains unicode characters. I removed it and now use a prebuilt version (2.4x) from [Graphviz Homepage](https://www.graphviz.org/download/) which works fine.
